example/linux/Makefile

```diff
@@ -20,6 +20,11 @@
 BINARY_NAME=flutter_desktop_example
 # The C++ code for the embedder application.
 SOURCES=flutter_embedder_example.cc
+# Extra flags (e.g., for library dependencies).
```

Comment:
The changes to this makefile are just because I like to minimize differences between the structure of `example` and `testbed`'s builds for clarity when people diff.